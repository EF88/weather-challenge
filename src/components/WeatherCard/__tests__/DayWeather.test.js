import React from 'react';
import { shallow } from 'enzyme';
import DayWeather from '../DayWeather';

describe('DayWeather', () => {

  const infoCity = {
    id: 5093304490262528,
    weather_state_name: "Heavy Cloud",
    weather_state_abbr: "hc",
    wind_direction_compass: "NE",
    created: "2020-03-23T10:36:49.295931Z",
    applicable_date: "2020-03-23",
    min_temp: 10.254999999999999,
    max_temp: 19.265,
    the_temp: 19.655,
    wind_speed: 5.0632496426071745,
    wind_direction: 46.385905633869974,
    air_pressure: 1013.5,
    humidity: 64,
    visibility: 13.390238507118427,
    predictability: 71
  }

  const setup = props => {
    return shallow(<DayWeather infoCity={ props } />);
  };

  it('shallow without fail', () => {
    setup(infoCity);
  });

  it('matches the snapshot', () => {
    const wrapper = setup(infoCity);
    expect(wrapper.html()).toMatchSnapshot();
  });
});
