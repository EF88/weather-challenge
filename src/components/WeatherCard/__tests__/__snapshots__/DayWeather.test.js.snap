// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DayWeather matches the snapshot 1`] = `"<div class=\\"sc-AxjAm DPHZE\\"><h4>Monday</h4><img src=\\"https://www.metaweather.com//static/img/weather/hc.svg\\" alt=\\"Smiley face\\" height=\\"42\\" width=\\"42\\"/><h1>20째</h1><hr class=\\"sc-AxirZ dmCkWA\\"/><div class=\\"sc-AxiKw hHZAiu\\"><div class=\\"sc-AxhCb hOiHna\\">min<div>10째</div></div><div class=\\"sc-AxhCb hOiHna\\">max<div>19째</div></div></div><div class=\\"sc-AxiKw hHZAiu\\"><div class=\\"sc-AxhCb hOiHna\\">wind speed<div>5째</div></div><div class=\\"sc-AxhCb hOiHna\\">wind dir<div>NE</div></div></div><hr class=\\"sc-AxirZ dmCkWA\\"/><h5>predictability</h5><h3>71%</h3></div>"`;
